src/main/java/edu/cornell/kfs/fp/businessobject/PurchasingDataBase.java

```diff
@@ -0,0 +1,525 @@
+      if (StringUtils.isNotBlank(freightShippingAmount)) {
+          this.freightShippingAmount = new KualiDecimal(freightShippingAmount);
+      } else {
+          this.freightShippingAmount = KualiDecimal.ZERO;
+      }
+    }
+
+
+    /**
+     * @param freightShippingAmount the freightShippingAmount to set
+     */
+    public void setFreightShippingAmount(KualiDecimal freightShippingAmount) {
+      this.freightShippingAmount = freightShippingAmount;
+    }
+
+
+    /**
+     * @return the dutyAmount
+     */
+    public KualiDecimal getDutyAmount() {
+      return dutyAmount;
+    }
+
+
+    /**
+     * @param dutyAmount the dutyAmount to set
+     */
+    public void setDutyAmount(KualiDecimal dutyAmount) {
+      this.dutyAmount = dutyAmount;
+    }
+
+
+    /**
+     * @param freightShippingAmount the freightShippingAmount to set
+     */
+    public void setDutyAmount(String dutyAmount) {
+      if (StringUtils.isNotBlank(dutyAmount)) {
+          this.dutyAmount = new KualiDecimal(dutyAmount);
+      } else {
+          this.dutyAmount = KualiDecimal.ZERO;
+      }
+    }
+
+
+    /**
+     * @return the destinationPostalZipCode
+     */
+    public String getDestinationPostalZipCode() {
+      return destinationPostalZipCode;
+    }
+
+
+    /**
+     * @param destinationPostalZipCode the destinationPostalZipCode to set
+     */
+    public void setDestinationPostalZipCode(String destinationPostalZipCode) {
+      this.destinationPostalZipCode = destinationPostalZipCode;
+    }
+
+
+    /**
+     * @return the shipFromPostalZipCode
+     */
+    public String getShipFromPostalZipCode() {
+      return shipFromPostalZipCode;
+    }
+
+
+    /**
+     * @param shipFromPostalZipCode the shipFromPostalZipCode to set
+     */
+    public void setShipFromPostalZipCode(String shipFromPostalZipCode) {
+      this.shipFromPostalZipCode = shipFromPostalZipCode;
+    }
+
+
+    /**
+     * @return the destinationCountryCode
+     */
+    public String getDestinationCountryCode() {
+      return destinationCountryCode;
+    }
+
+
+    /**
+     * @param destinationCountryCode the destinationCountryCode to set
+     */
+    public void setDestinationCountryCode(String destinationCountryCode) {
+      this.destinationCountryCode = destinationCountryCode;
+    }
+
+
+    /**
+     * @return the uniqueVATInvoice
+     */
+    public String getUniqueVATInvoice() {
+      return uniqueVATInvoice;
+    }
+
+
+    /**
+     * @param uniqueVATInvoice the uniqueVATInvoice to set
+     */
+    public void setUniqueVATInvoice(String uniqueVATInvoice) {
+      this.uniqueVATInvoice = uniqueVATInvoice;
+    }
+
+
+    /**
+     * @return the orderDate
+     */
+    public Date getOrderDate() {
+      return orderDate;
+    }
+
+
+    /**
+     * @param orderDate the orderDate to set
+     */
+    public void setOrderDate(Date orderDate) {
+      this.orderDate = orderDate;
+    }
+    
+
+    public void setOrderDate(String orderDate) {
```

Comment:
This setter will only overwrite the orderDate if the String is non-blank. Is this intentional, or should it allow for setting the orderDate to null for a blank String?